gammapy/data/observations.py

```diff
@@ -910,3 +915,58 @@ def check_psf(self):
         except Exception:
             yield self._record(level="warning", msg="Loading psf failed")
             return
+
+
+def create_observation_from_fermi_files(
+    path,
+    events_filename="ft1_00.fits",
+    counts_filename="ccube_00.fits",
+    exposure_filename="bexpmap_roi_00.fits",
+    psf_filename="psf_00.fits",
+    drm_filename="drm_00.fits",
+    obs_id=0,
+):
+    """Read Fermi-LAT files and create an Observation.
+
+    Parameters
+    ----------
+    path : str
+        path to files
+    events_filename : str
+        Events filename. Default is ft1_00.fits
+    counts_filename : str, optional
+        Counts filename. Default is ccube_00.fits
+    exposure_filename : str, optional
+        exposure filename. Default is bexpmap_roi_00.fits
+    psf_filename : str, optional
+        PSF filename. Default is psf_00.fits
+    drm_filename : str, optional
+        DRM filename. Default is drm_00.fits
+    obs_id : int, optional
+        Observation ID. Defalut is 0.
+
+    Returns
+    -------
+    observation : `~gammapy.data.Observation`
+        Observation with DL4 IRFs
+    """
+
+    path = Path(path)
+
+    events = EventList.read(path / events_filename)
+    counts = Map.read(path / counts_filename)
```

Comment:
So you dont really need the counts, only a geom?

Reply:
yes, counts are used only to get the `energy_axis` and check that the drm was generated with the same `edisp_bins` option than the exposure/counts (not by the default).
Note also that the counts map created by fermipy is by default in a different projection than exposure (AIT vs CAR) so we cannot use them as such in gammapy.

Reply:
This is also I question I had. In the gammapy tuto for Fermi, the counts cube is barely used and recreated from event list and I did not know why.
So you say it's related to this projection issue ?